src/main/java/gov/cms/madie/util/HumanReadableUtil.java

```diff
@@ -126,7 +128,11 @@ public static String getPopulationDescription(Measure measure, String population
                 }
               });
     }
-    return sb.toString();
+    if (sb.toString().length() == 1 && sb.toString().contains("\n")) {
+      return "None";
+    } else {
+      return sb.toString();
+    }
```

Comment:
If the concern is whether the resulting String will only be whitespace, then recommend StringUtils.isBlank():
```suggestion
    if (StringUtils.isBlank(sb.toString())) {
      return "None";
    } else {
      return sb.toString();
    }
```
See: [isBlank](https://commons.apache.org/proper/commons-lang/apidocs/org/apache/commons/lang3/StringUtils.html#isBlank-java.lang.CharSequence-).

Reply:
Mimicking the return style in [MappingUtil](https://github.com/MeasureAuthoringTool/madie-qdm-service/pull/56/files#diff-61e99139e9eabdfcc7c7f34ffffc632cf3ffe78228401dc279c71f5c83e14753R111) would also be accepted.

Reply:
oh, duh. fixed, thank you